lib/improver/nowcasting/forecasting.py

```diff
@@ -420,3 +421,24 @@ def extrapolate(self, leadtime_minutes=None):
                 forecast_cube, self.orographic_enhancement_cube)
 
         return forecast_cube
+
+    def process(self, interval, max_lead_time):
```

Comment:
Have you chosen not to write unit tests for this method because it is essentially a wrapper for another method?

Reply:
Yes.  All I've done is pull the lead time loop out of CLIs and into this wrapper.  There are unit tests for the extrapolate function, and CLI tests to cover the loop, so I didn't see a good reason to cover this further.  (Also the likelihood is that we will be retiring this plugin altogether and replacing it with the pysteps extrapolation function.)

Reply:
I'm not entirely comfortable with relying on CLI tests to cover plugin functionality, but I'm not going to ask you to write a unit test for this now.